src/Microsoft.Health.Fhir.Liquid.Converter/Filters/GeneralFilters.cs

```diff
@@ -73,6 +73,8 @@ public static string GenerateUUID(string input)
             var guid = new byte[16];
             Array.Copy(hash, 0, guid, 0, 16);
             return new Guid(guid).ToString();
+
         }
+
```

Comment:
The empty lines could be removed from this file.

Reply:
fixed in the update.